crates/transaction-fuzzer/src/account_universe/transfer_gen.rs

```diff
@@ -232,9 +232,7 @@ pub fn gas_price_selection_strategy() -> impl Strategy<Value = u64> {
 pub fn gas_budget_selection_strategy() -> impl Strategy<Value = u64> {
     prop_oneof![
         Just(0u64),
-        Just(PROTOCOL_CONFIG.base_tx_cost_fixed() - 1),
-        Just(PROTOCOL_CONFIG.base_tx_cost_fixed()),
-        Just(PROTOCOL_CONFIG.base_tx_cost_fixed() + 1),
+        PROTOCOL_CONFIG.base_tx_cost_fixed() / 2..=PROTOCOL_CONFIG.base_tx_cost_fixed() * 2000,
```

Comment:
@tzakian take a look here, minimum cost is now a variable of the gas price so we cannot make it a hard number.
I think the logic here is still fine as it will pick a random number in a range that is good or bas and it's the same logic as before. Before it was very well defined and precise, now we cannot